openedx/features/caliper_tracking/tests/expected/edx.course.enrollment.activate.json

```diff
@@ -0,0 +1,41 @@
+{
+  "@context": "http://purl.imsglobal.org/ctx/caliper/v1p1",
+  "id": "urn:uuid:2635b9dd-0061-4059-ac61-2718ab366f75",
+  "type": "Event",
+  "actor": {
+    "id": "edx.devstack.lms:18000/u/ali",
```

Comment:
user `ali` does not exist in the current file. Please use the username from that file

Reply:
fixed